subsys/net/lib/lwm2m/lwm2m_object.h

```diff
@@ -773,6 +765,16 @@ static inline int engine_put_corelink(struct lwm2m_output_context *out,
 	return -ENOTSUP;
 }
 
+static inline int engine_put_timestamp(struct lwm2m_output_context *out, int64_t timestamp)
+{
+	if (out->writer->put_data_timestamp) {
+		return out->writer->put_data_timestamp(out, timestamp);
+	}
+
+	return 0;
```

Comment:
```suggestion
	return -ENOTSUPP;
```
or something.. it should be error to call this function for content formats that don't support it.